resolwe_bio/docker_images/rnaseq/packages-debian.txt

```diff
@@ -19,3 +19,6 @@ libssl-dev
 python3-numpy
 python3-pandas
 python3-sklearn
+
+# required for jbrowse
+# libbio-perl-perl
```

Comment:
You probably wish to un-comment this.

Reply:
Actually, this needs to be removed. Thanks.